### DISCUSSION TITLE
When I switch networks the app crashs.

### DISCUSSION THREAD

**Discussion by ssrlive** [libuv/libuv#3292] [Q&A]
Every time I switch from Wi-Fi network 1 to Wi-Fi network 2, the program crashes. 

How can I avoid this crash? 

Will the event loop give the user an opportunity to handle the cleanup procedure and start a new loop when switching networks?

**Comment by bnoordhuis** [comment]
Can you define "crash"? Do you mean it segfaults, libuv functions start returning errors (and if so, what errors), or something else?
